cmd/tailscale/cli/debug.go

```
@@ -807,6 +819,46 @@ func runDaemonBusEvents(ctx context.Context, args []string) error {
 	return nil
 }
 
+var daemonBusGraphArgs struct {
+	format string
+}
+
+func runDaemonBusGraph(ctx context.Context, args []string) error {
+	graph, err := localClient.EventBusGraph(ctx)
+	if err != nil {
+		return err
+	}
+	if daemonBusGraphArgs.format == "dot" {
+		var topics eventbus.DebugTopics
+		json.Unmarshal(graph, &topics)
```

Although it is unlikely, we should report an error if unmarshaling fails.